app/assets/stylesheets/user/profiles.scss

```diff
@@ -0,0 +1,12 @@
+.user-profiles {
+  &.edit {
+    .image > img {
+      height: 200px;
+      width: 200px;
+    }
+
+    input#user_profile_photo {
```

Comment:
Actually this would always be a false positive.

Can you add a separate ID or class to the element directly to use for styling? Rails might change the generated IDs also.